### PR TITLE
Feature/OTC-212 add extension field to party info

### PR DESCRIPTION
**Pull request by lazolalucas**

### CODE REVIEW

**Review by ndonnan — APPROVED** [review]
Looks good

**Review by ndonnan — CHANGES_REQUESTED** [review]
Please bump version number

File: src/test-api/api.yaml [hunk 1]
```diff
@@ -139,19 +159,40 @@ components:
...
+            - ACCOUNT_ID
+            - IBAN
+            - ALIAS
         idValue:
           type: string
           minLength: 1
           maxLength: 128
+        extensionList:
+          $ref: '#/components/schemas/ExtensionList'
+    Extension:
```
Comment (rmothilal) [inline src/test-api/api.yaml:176]: @lazolalucas  you have added Extensions and ExtensionList but i don't see it being referenced anywhere in the yaml??

**Review by rmothilal — CHANGES_REQUESTED** [review]
Please see comment

File: src/test-api/api.yaml (hunk 1 above)
Comment (lazolalucas) [reply src/test-api/api.yaml]: Yes thanks I forgot to remove this

Comment (lazolalucas) [reply src/test-api/api.yaml:176]: Extension is the object referenced in ExtensionList the array which in turn referenced in the party component

Comment (lazolalucas) [reply src/test-api/api.yaml:176]: Extension is the object referenced in ExtensionList the array which in turn referenced in the party component

**Review by rmothilal — APPROVED** [review]
Happy

**Review by ndonnan — APPROVED** [review]

**Pull request merged**